Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -47,6 +47,16 @@
     - id: SmokingPipeFilledTobacco
     - id: FlippoLighter #Not the steal objective, only difference from normal detective trenchcoat
 
+- type: entity
```

Comment:
I couldn't be bothered making a proper sprite ok if someone wants to do it just do it I'm not your mom I'm just very tired